public/build/css/all.css

```diff
@@ -1,3 +1,7 @@
+/*!
```

Comment:
Perhaps @joecohens is compiling with `gulp --production`? I should be doing that, in fact, it should be default.